src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -126,7 +127,7 @@ export default {
 	computed: {
 		showAppeal() {
 			// Check if Appeal Banner is active and the user is not on a denied page URL
-			if (this.appealEnabled && !isExcludedUrl(globalBannerDenyList, this.$route.path)) return true;
+			if (this.appealEnabled && !isExcludedUrl(globalBannerDenyList, [], this.$route.path)) return true;
```

Comment:
isExcludedUrl will be called already in the prefetch result, so `showAppeal` can just be `return this.appealEnabled;` or the usages of it could be replaced with appealEnabled.